apps/test/unit/lib/levelbuilder/script-editor/ScriptEditorTest.jsx

```diff
@@ -222,6 +224,30 @@ describe('ScriptEditor', () => {
     });
   });
 
+  it('disables peer review count when instructor review only selected', () => {
+    const wrapper = createWrapper({
+      initialOnlyInstructorReviewRequired: false,
+      initialPeerReviewsRequired: 2
+    });
+
+    let peerReviewCountInput = wrapper.find('#number_peer_reviews_input');
+
+    expect(peerReviewCountInput.props().disabled).to.be.false;
+    expect(peerReviewCountInput.props().value).to.equal(2);
+
+    const instructorReviewOnlyCheckbox = wrapper.find(
+      '#only_instructor_review_checkbox'
+    );
+    instructorReviewOnlyCheckbox.simulate('change', {
+      target: {checked: true}
+    });
+
+    peerReviewCountInput = wrapper.find('#number_peer_reviews_input');
```

Comment:
Is it necessary to query for the input element again, or does the test still pass if you use the element assigned on like 233?

Reply:
I had to query it again, which I was surprised by. Tried wrapper.update(), peerReviewCountInput.update() with no luck.